app/assets/stylesheets/image_x.scss

```diff
@@ -104,3 +104,32 @@ $image-thumb-slider-background-color: $sul-background;
   white-space: nowrap;
   width: 100%;
 }
+
+.#{$namespace}-image-x-side-control {
+  bottom: 0;
+  color: white;
+  cursor: pointer;
+  font-size: 2em;
+  left: 0;
+  position: absolute;
+  text-align: center;
+  text-shadow: 0 1px 2px rgba(0,0,0,.6);
```

Comment:
`.6` should be written with a leading zero as `0.6`<br>Commas in function arguments should be followed by a single space